### PR TITLE
Use chooser for sharing logs

### PR DESCRIPTION
**Pull request by NightlyNexus**

### CODE REVIEW

File: src/main/java/com/jakewharton/u2020/util/Intents.java [hunk 1]
```diff
@@ -15,7 +15,22 @@
    * will display a simple message if none are available to handle it.
    */
   public static boolean maybeStartActivity(Context context, Intent intent) {
+    return maybeStartActivity(context, intent, false);
+  }
+
+  /**
+   * Similar to {@link #maybeStartActivity(Context, Intent)} but launches Android's chooser when
+   * there are multiple resolutions
```
Comment (JakeWharton) [inline src/main/java/com/jakewharton/u2020/util/Intents.java]: Technically not true. It launches the chooser should any handlers be available.

Also you forgot a period on your comment. [Comments should be sentences][1]

 [1]: nedbatchelder.com/blog/201401/comments_should_be_sentences.html

**Comment by JakeWharton** [comment]
lgtm

File: src/main/java/com/jakewharton/u2020/util/Intents.java (hunk 1 above)
Comment (NightlyNexus) [inline src/main/java/com/jakewharton/u2020/util/Intents.java]: I am 95% sure the chooser is bypassed when there is only one handler, though.  I will check.
I have actually been meaning to ask about comments and logs with full stops; glad I made that error haha.  Thanks for the link!
(The link above is mashed with another url, so here it is correctly for posterity: http://nedbatchelder.com/blog/201401/comments_should_be_sentences.html)

Comment (NightlyNexus) [inline src/main/java/com/jakewharton/u2020/util/Intents.java]: Yeah, the chooser is bypassed, but, technically, yes, the chooser is still called.  I'm updating the doc.

**Pull request merged**